doc/modules/changes/20190505_gassmoeller

```diff
@@ -0,0 +1,7 @@
+Changed: The lateral averaging functions, e.g. used in the 'depth average'
+postprocessor were optimized for speed. Their lateral averaging is now slightly
+less accurate (the difference is typically on the order of 1e-3 at the coarsest
```

Comment:
`less accurate` ... `for specific geometry models (Spherical Shell, Box, Two Merged Boxes)`